### PR TITLE
OCLOMRS-2: User dashboard after login for dictionary management (UI)

### PR DESCRIPTION
**Pull request by shakirandagire**
# JIRA TICKET NAME:
[OCLOMRS-2: User dashboard after login for dictionary management (UI)](https://issues.openmrs.org/browse/OCLOMRS-2)

# Summary:
After a successful login attempt, a user should be redirected to their dashboard (Homescreen) where they can view their preferred openMRS dictionaries.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -85,4 +85,52 @@
     color: #ccc !important;
     background: none;
   }
+
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
+
+p{
```
Comment (fuchodeveloper) [inline src/App.css]: How about creating a class to target this particular element instead of implementing the style for all `p` elements?

Comment (shakirandagire) [reply src/App.css]: Thank you. Let me work on that.

**Comment by shakirandagire** [comment]
@geofrocker Let me put them in gitignore

**Comment by shakirandagire** [comment]
@fuchodeveloper That was not intentional and I rectified it when I squashed.

**Comment by geofrocker** [comment]
@shakirandagire Travis creates its own snapshots when running the tests, so the coverage percentage won't be affected. This will also help in reducing merge conflicts in future as the project grows. However, this is an issue you can sync on with your team members

**Comment by shakirandagire** [comment]
@geofrocker Thanks again. Let me sync with my team and see their feedback on the matter

File: src/App.css (hunk 1 above)
Comment (fuchodeveloper) [reply src/App.css]: Awesome 👍

**Comment by shakirandagire** [comment]
@geofrocker I have had a sync with team and we have agreed not to push the snapshots as you had suggested before.

**Comment by mwaz** [comment]
did you rebase before making the PR ?

**Comment by shakirandagire** [comment]
@mwaz ,Yes I did, but when I squashed I got all these files that had changed.

**Comment by shakirandagire** [comment]
@mwaz I have resolved the issues with the changed files.

**Comment by mwaz** [comment]
Also, remove the empty Snapshots file as discussed.

**Comment by shakirandagire** [comment]
@mwaz The deleted snapshot was from @hadijahkyampeire's merged PR and the effect will be reflected when my PR is merged

**Pull request merged**